src/content/use-cases/object-detection/sparsifying.mdx

```diff
@@ -44,9 +42,9 @@ sparseml.yolov5.train \
   --recipe zoo:cv/detection/yolov5-s/pytorch/ultralytics/coco/pruned_quant-aggressive_94
 ```
 
-- `--weights` argument indicates which model to start the pruning process from. It can be a SparseZoo stub or a local path to a model.
-- `--data` specifies the dataset to be used. You may sparsify your model while training on your own, private (downstream) dataset or while continuing training with the original (upstream) dataset. The [configuration file](https://github.com/neuralmagic/sparseml/blob/main/src/sparseml/yolov5/data/coco.yaml) for COCO is included in the yolov5 integration and can be used as an example for a custom dataset.
-- `--recipe` encodes the hyperparameters of the pruning process. It can be a SparseZoo stub or a local YAML file. See [here](/user-guide/recipes/creating) for a detailed discussion of recipes.
+- `--weights` indicates the model with which to start the pruning process. It can be a SparseZoo stub or a local path to a model.
```

Comment:
```suggestion
- `--weights` indicates the checkpoint from which the pruning process should start. It can be a SparseZoo stub or a local path to a model.
```